### DISCUSSION TITLE
My output variable 'LH' is invalid

### DISCUSSION THREAD

**Discussion by FuYingWang** [NCAR/hrldas#141] [Q&A] [answered]
I used the data provided in tutorial 3 to run the model, but there is an issue with the variable 'LH' value in the output file.
![down](https://github.com/NCAR/hrldas/assets/153582912/d142c69c-5a61-4927-b4d9-d55439b6229a)
The namelist.hrldas like that.
![namelist1](https://github.com/NCAR/hrldas/assets/153582912/57929be9-8c43-49a9-8583-b2a6c97a97cc)
![参数化方案](https://github.com/NCAR/hrldas/assets/153582912/a3f3e3fe-db22-4d7b-8086-470cdfdbeff0)
![name3](https://github.com/NCAR/hrldas/assets/153582912/3c8990b5-bab4-4a57-970a-6d4948bca4c9)
Then, I got output file like that.
![outputfile](https://github.com/NCAR/hrldas/assets/153582912/77e34e9f-c25e-487b-9a09-093d49650959)
ncview one file.
![ncview](https://github.com/NCAR/hrldas/assets/153582912/8eda28ab-2e4b-4400-a92b-4c17ce95efbe)
Some variable such as 'LAI','SOIL_M' is available
LAI:   ![lai](https://github.com/NCAR/hrldas/assets/153582912/417e779b-54e6-4d74-a41d-dd3ce9ec6ce2)
SOIL_M:   
![soilM](https://github.com/NCAR/hrldas/assets/153582912/ca02e9b9-9048-4bd2-a913-bd177d186c98)

But there seems to be something wrong with variable 'LH' 
![LH](https://github.com/NCAR/hrldas/assets/153582912/f86b6289-0663-4bd6-887f-9ad6856c6a4f)
Do you know why this is? Did I make a mistake in one step？
Thanks

**Comment by cenlinhe** [comment]
can you change the color range for your "LH" variable to be something like (-50 ~ 50) in your ncview window?

**Reply by cenlinhe** [reply, marked as answer]
just click the "Range" item and change the minimum and maximum values to be -50 and 50

**Reply by FuYingWang** [reply]
Oh!!
It seem ok
![image](https://github.com/NCAR/hrldas/assets/153582912/b2b49220-330e-49d8-8396-454e2d41fabd)

**Reply by FuYingWang** [reply]
Thank

**Reply by cenlinhe** [reply]
yes, it looks reasonable. The red part is just for lake points.
